### DISCUSSION TITLE
Webhook을 이용해 Discord에 알림 보내는 방법을 공유합니다!

### DISCUSSION THREAD

**Discussion by JIWON1923** [Team-ShortcutsZip/MacC-Team-HappyAnding#392] [General]
1. Discord 서버 설정으로 들어갑니다.
   (권한이 없는 경우 서버 생성자에게 요청하면 됩니다!)
![image](https://user-images.githubusercontent.com/68676844/217809254-461c7931-9ce8-40b6-99c4-fdd0665bda9b.png)

2. 연동 탭에서 웹 후크로 들어갑니다.

3. 새 웹 후크 버튼을 클릭하고 채워줍니다.
  (이름: 웹 후크 이름으로, 제가 웹 후크를 구분하기 위해 필요한 용도입니다. 채널은 웹 후크로 들어온 이벤트를 어떤 채널에 보여줄지 결정하는 것입니다)
![image](https://user-images.githubusercontent.com/68676844/217809433-b503ade7-a965-44c1-88b4-de845a347c8b.png)

4. 웹후크 URL 복사를 클릭해주세요

5. 깃허브 setting 탭에서 웹후크에 들어가주세요 -> add web hook 버튼 클릭!
![image](https://user-images.githubusercontent.com/68676844/217809819-c5545199-1e75-4130-b221-7cb6e91b8ce6.png)

6. 헤당 필드를 채워주면 됩니다.
 - Payload URL: 복사해준 웹후크 URL을 복붙해줍니다 (주의!! **맨 뒤에 /github 를 꼭 붙여주세요**)
 - content는 json으로 받아줍니다.
 - 저는 알림의 종류를 채널로 분리하고 싶어서 individual event로 바꿔줬습니다.
![image](https://user-images.githubusercontent.com/68676844/217810095-d4e7ec5a-91d0-49df-9c59-d1b7fe916717.png)

![image](https://user-images.githubusercontent.com/68676844/217810381-632fc68d-a51a-44c0-a03e-16e72849ee71.png)

**Comment by HanGyeongjun** [comment]
오오 thirdparty 프로그램을 이용하신 줄 알았는데 의외로 github 자체적으로 지원하는 기능이었네요! 공유 감사합니다 💯
